jupyterlab/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM jupyter/tensorflow-notebook:latest
+
+USER $NB_UID
+
+# Conda packages
+RUN conda install --quiet --yes \
+    'nb_conda_kernels' && \
```

Comment:
I don't think this needs to be in quotes.

Reply:
Agreed